README.md

```diff
@@ -101,6 +101,14 @@ Install the latest stable version of DraNet using the provided manifest:
 kubectl apply -f https://raw.githubusercontent.com/google/dranet/refs/heads/main/install.yaml
 ```
 
+### Development
+
+To build your own image for testing. Here is an example with a custom registry `ghcr.io/converged-computing`:
```

Comment:
This is only necessary when developing on "real" clusters, right?  We might clarify that.

A lot of this is developed locally, where the image name is ~arbitrary, because we don't push it to a remote host anyhow.